src/browser.ts

```diff
@@ -7,6 +7,7 @@ import { WEBSOCKET_ENDPOINT_REGEX, websocketReady } from "./util.ts";
 
 async function runCommand(
   command: Deno.Command,
+  { windowsErr21RetryAttempts = 60 } = {},
```

Comment:
I'm not sure how I feel about this naming. Maybe we could just call this "retries"? I think this could be useful throughout actually (even in the `SingletonLock` case).